pkg/statistics/handle/ddl/reorganize_partition.go

```diff
@@ -15,10 +15,10 @@
 package ddl
 
 import (
-	"github.com/pingcap/tidb/pkg/statistics/handle/util"
+	ddlutil "github.com/pingcap/tidb/pkg/ddl/util"
```

Comment:
maybe move those structs to a named pkg later, such as `schemachange` or `evnets`, `util` mostly means something more common

Reply:
Yes, will do it in future